object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,29 @@
+/* exported omit */
+
+// create storage for output
+// create storage for test if a key isn't in the source, set as true
+// look at each key in source argument
+// - look at each element in keys argument
+//  - if current element is equal to current key, set key test to false
+// - if key test is true
+//  - add current key to output
+// - otherwise, set key test to true
```

Comment:
Again, I'm not sure what this means. Try to make sure your pseudocode avoids tech jargon and fully describes its purpose and what the line is meant to do.